contracts/Strategy.sol

```diff
@@ -192,21 +269,17 @@ contract Strategy is BaseStrategy {
                 address(this),
                 address(this),
                 wantBalance,
-                0 // setting this to 0, let's the previous argument determine the deposit size
+                0 // setting this to 0 lets the previous argument determine the wantBalance size
             );
         }
 
         uint256 sharesInBento = sharesInBento();
 
         if (sharesInBento > wantToBentoShares(dustThreshold, false)) {
-            bentoBox.transfer(
-                BIERC20(address(want)),
-                address(this),
-                address(kashiPair),
-                sharesInBento
-            );
+            // Get highest interest rate pair
+            uint256 highestInterestIndex = highestInterestPairIndex();
```

Comment:
is it possible / worth it to calculate what is the highestInterestPairIndex taking into account the amount that will be deposited? 

i mean, calculating the interest that will be paid if we deposit that amount. maybe a lower volume pair is impacted higher by a deposit, making the interest paid plunge

Reply:
Yup, i should see if adapting this is worthwhile: https://github.com/sushiswap/kashi-lending/blob/master/contracts/KashiPair.sol#L187-L205

Reply:
Hmm, i'm less certain now that i see that the target utilization parameters aren't exposed. https://github.com/sushiswap/kashi-lending/blob/master/contracts/KashiPair.sol#L112-L113

Reply:
@jmonteer https://github.com/fp-crypto/kashi-strategy/pull/2/commits/6e4225d3716ddc3dd73107972c94b66206f3bf22 uses the new deposit and a more accurate model, mind taking a look?

Reply:
reviewing